.env.test

```diff
@@ -1,11 +1,15 @@
+# Fake mongo env variable
+MONGODB_CONNECTION_STRING=mongodb://fakemongo.com
+DATABASE_NAME=FAKEDATABASE_NAME
+
 # Flask server configuration.
-FLASK_APP=todo_app/app
+FLASK_APP=todo_app.app:create_app
```

Comment:
I don't think this variable actually matters in `.env.test` since we're not expecting Flask to work out where to start the app, but worth noting this shouldn't change any behaviour: flask will automatically look for a `create_app` function even if not specified. That said, that also means absolutely no harm in spelling it out this way if you prefer it!